lib/smart_answer/calculators/pay_leave_for_parents_calculator.rb

```diff
@@ -14,6 +14,7 @@ class PayLeaveForParentsCalculator
     attr_accessor :salary_1_66_weeks
     attr_accessor :partner_started_working_before_continuity_start_date
     attr_accessor :partner_still_working_on_continuity_end_date
+    attr_accessor :partner_earned_more_than_lower_earnings_limit
```

Comment:
Is this used anywhere? It seems odd that no other code in the flow or template has had to change.